src/core/Form/TextInput/TextInput.md

```diff
@@ -12,7 +12,7 @@ import { TextInput } from 'suomifi-ui-components';
 
 <>
   <TextInput
-    onBlur={event => console.log(event.target.value)}
+    onBlur={(event) => console.log(event.target.value)}
```

Comment:
Some console logs seem to be empty in styleguidist. Needs to be fixed?

Reply:
Nevermind, as it is onBlur, the event fires even if the value of the input has not changed when we navigate away from the input.